### PR TITLE
Fix BTCETH price feed

### PR DESCRIPTION
**Pull request by rawfalafel**

### CODE REVIEW

File: v0/scripts/PublishPrices.js [hunk 1]
```diff
@@ -12,7 +12,11 @@ const alphaVantageKey = process.env.ALPHAVANTAGE_API_KEY;
 const barchartKey = process.env.BARCHART_API_KEY;
 
 function stripApiKey(str, key) {
-  return str.replace(key, "{redacted}");
+  if (key) {
+    return str.replace(key, "{redacted}");
```
Comment (rawfalafel) [inline v0/scripts/PublishPrices.js]: Its possible for `key` to be an empty string. Probably a cleaner way to address this, but this seemed the easiest for now.

Comment (EdouardLvdl) [reply v0/scripts/PublishPrices.js]: thanks for the fix

File: v0/scripts/PublishPrices.js [hunk 2]
```diff
@@ -74,7 +78,7 @@ async function getBarchartPrice(asset) {
 
 // Gets the Coinbase price for an asset or throws.
 async function getCoinbasePrice(asset) {
-  const url = `https://api.coinbase.com/v2/prices/${asset}/spot`;
+  const url = `https://api.coinbase.com/v2/prices/${asset}-USD/spot`;
```
Comment (ptare) [inline v0/scripts/PublishPrices.js]: Is this change incompatible with PR #430, not sure which approach we prefer.

Comment (mrice32) [reply v0/scripts/PublishPrices.js]: I think I prefer #430 since it makes the config more explicit.

**Comment by mrice32** [comment]
Can you merge master into your branch?

**Comment by rawfalafel** [comment]
going to close this PR and open with a new source branch.

**Pull request closed without merging**